feature-libs/product-configurator/rulebased/core/state/effects/configurator-cart.effect.spec.ts

```diff
@@ -307,27 +306,55 @@ describe('ConfiguratorCartEffect', () => {
         userId: userId,
         cartId: cartId,
         productCode: payloadInput.productCode,
-        quantity: cartModification.quantity,
-        deliveryModeChanged: cartModification.deliveryModeChanged,
-        entry: cartModification.entry,
-        quantityAdded: cartModification.quantityAdded,
-        statusCode: cartModification.statusCode,
-        statusMessage: cartModification.statusMessage,
+        quantity: 1,
+        deliveryModeChanged: true,
+        entry: entry,
+        quantityAdded: 1,
+        statusCode: emptyStatus,
+        statusMessage: emptyStatus,
       });
 
       const addNextOwner = new ConfiguratorActions.AddNextOwner({
         ownerKey: owner.key,
         cartEntryNo: '' + entryNumber,
       });
-      actions$ = hot('-a', { a: action });
+      actions$ = cold('-a', { a: action });
```

Comment:
just curious: why we're changing those all hot to cold?

Reply:
Only because it's not needed and confusing (it's weird that the input observables are hot and the expected result is cold. Although the tests pass. But someone looking at the tests can think that there's a specific meaning to the difference, like one colleague did last week)